lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -47,8 +47,8 @@ module.exports = class MethodMappings {
         }
       },
 
-      getSessions() {
-        return '/sessions';
+      async getSessions() {
```

Comment:
this is a legacy JSONWiere endpoint and we need to keep it for backwards compatibility only.